packages/@stimulus/core/src/target_properties.ts

```diff
@@ -33,6 +33,9 @@ function propertiesForTargetDefinition(name: string) {
       get(this: Controller) {
         return this.targets.has(name)
       }
-    }
+    },
+
+    [`${name}TargetAdded`]: (element: Element) => {},
+    [`${name}TargetRemoved`]: (element: Element) => {},
```

Comment:
@sstephenson I've added these blessings, but they might not be necessary. What can I do to minimize the friction with the class-level property interface work that's in flight?

Reply:
Don't worry about what's in flight. But also, these additions aren't necessary. You're already checking whether the callback exists before you invoke it.